zgrab_schema.py

```diff
@@ -202,7 +202,8 @@
     }),
     "session_ticket":SubRecord({
         "value":Binary(),
-        "length":Integer()
+        "length":Integer(),
```

Comment:
We usually only store length (in bits) for primes and keys. We've removed it in other places, such as the ZMap JSON output.